### PR TITLE
Sample 3ds new flow

### PR DESCRIPTION
**Pull request by omarxp**
move authentication (3DS) param to backend

### CODE REVIEW

File: examples/notification-handler.php [hunk 1]
```diff
@@ -1,12 +1,31 @@
...
+Config::$serverKey = '<your server key>';
+if (strpos(Config::$serverKey, 'your ') != false ) {
+    echo "<code>";
+    echo "<h4>Please set your server key from sandbox</h4>";
+    echo "In file: " . __FILE__;
+    echo "<br>";
+    echo "<br>";
+    echo htmlspecialchars('Config::$serverKey = \'<your server key>\';');
+    die();
+}
```
Comment (rizdaprasetya) [inline examples/notification-handler.php:19]: This block is not part of the example. Can be confusing to read this part of code. Can you make it a function that is defined in a bottom?

e.g. at the bottom of  this file:
```
// non-relevant function only used for demo/example purpose
function printExampleWarningMessage(){
...
}
```
Then call it on line 11.

Also within that function check if method is not POST, then print warning: 
Notification-handler are not meant to be opened via browser / GET HTTP method. It is used to handle Midtrans HTTP POST notification / webhook. <put link to docs about notif handling>

File: examples/snap-redirect/checkout-process.php [hunk 2]
```diff
@@ -1,10 +1,23 @@
...
+Config::$serverKey = '<your server key>';
+if (strpos(Config::$serverKey, 'your ') != false ) {
+    echo "<code>";
+    echo "<h4>Please set your server key from sandbox</h4>";
+    echo "In file: " . __FILE__;
+    echo "<br>";
+    echo "<br>";
+    echo htmlspecialchars('Config::$serverKey = \'<your server key>\';');
+    die();
+}
```
Comment (rizdaprasetya) [inline examples/snap-redirect/checkout-process.php:20]: refactor as function `printExampleWarningMessage()`. Also applies to other file that have this codeblock

File: examples/core-api/checkout.php [hunk 3]
```diff
@@ -157,12 +134,60 @@ function closeDialog() {
...
                 event.preventDefault();
                 $(this).attr("disabled", "disabled");
+
+                console.log('1. get token_id');
                 MidtransNew3ds.getCardToken(card, callback);
                 return false;
             });
+
+            // callback functions
+            var callback = {
```
Comment (rizdaprasetya) [inline examples/core-api/checkout.php:153]: better let's rename this var to `getCardTokenCallback` for more descriptive code

File: examples/core-api/checkout.php [hunk 4]
```diff
@@ -1,17 +1,22 @@
...
 // can find in Merchant Portal -> Settings -> Access keys
-Config::$clientKey = "<your client key>";
-
+Config::$clientKey = '<your client key>';
 if (strpos(Config::$clientKey, 'your ') != false ) {
-    echo "<p style='background: #FFB588; padding: 10px;'>";
-    echo "Please set your client key in file " . __FILE__;
-    echo "</p>";
+    echo "<code>";
+    echo "<h4>Please set your server key from sandbox</h4>";
```
Comment (rizdaprasetya) [inline examples/core-api/checkout.php:14]: message should be `client key`, not `server key`. Also refactor as `printExampleWarningMessage` function

**Review by rizdaprasetya — CHANGES_REQUESTED** [review]
👍 nice. but since the example code is old and not updated for a long time, I think we can improve this. Please follow the commented suggestion. Thanks @omarxp

**Comment by omarxp** [comment]
@rizdaprasetya add a new commit for PR feedback

**Review by rizdaprasetya — APPROVED** [review]
Nice, looks good now 👍 please go ahead merge. Thanks @omarxp !

**Pull request merged**